examples/gno.land/r/demo/boards/z_4_filetest.gno

```diff
@@ -959,6 +959,7 @@ func main() {
 //     }
 // }
 // switchrealm["gno.land/r/demo/boards"]
+// switchrealm["gno.land/r/demo/boards"]
```

Comment:
I'm not sure, but I'm guessing it's because the `formatTypeValue` method doesn't handling some features properly. I think that integrating this `formatTypeValue()` into `TypeValue.Sprint` might solve it.

I also think the additional points you made below are related too.